<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-I3V0BKGE" name="com.buddi.colombiaapp.tests.LoginColombiaAppTest" tests="5" failures="0" timestamp="06 Sep 2020 14:18:50 GMT" time="24.018" errors="1">
  <testcase name="getAppHomeScreenMainTitle" time="0.268" classname="com.buddi.colombiaapp.tests.LoginColombiaAppTest"/>
  <testcase name="stopAppiumServer" time="0.579" classname="com.buddi.colombiaapp.tests.LoginColombiaAppTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.buddi.colombiaapp.tests.AppiumServerJava.stopServer(AppiumServerJava.java:44)
	at com.buddi.colombiaapp.tests.AppiumBaseTest.stopAppiumServer(AppiumBaseTest.java:118)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:350)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- stopAppiumServer -->
  <testcase name="getAppHomeScreenSubTitle" time="0.146" classname="com.buddi.colombiaapp.tests.LoginColombiaAppTest"/>
  <testcase name="loginColombiaApp" time="15.256" classname="com.buddi.colombiaapp.tests.LoginColombiaAppTest"/>
  <testcase name="logOutColombiaApp" time="7.769" classname="com.buddi.colombiaapp.tests.LoginColombiaAppTest"/>
</testsuite> <!-- com.buddi.colombiaapp.tests.LoginColombiaAppTest -->
